CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.2.0] - 2020-01-02
```

Comment:
Hey folks, I just want to call this version bump out for the person building the artifact - as, depending on if you're using git-hooks or some other process, this might need to be configured upon release. I chatted with a coworker and we had felt that because this release adds support for a major language version in a backwards compatible way, we'd bump the minor version instead of the patch. If you feel this is more of a "patch-worthy" change, I'll change this line right here.